src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -264,22 +264,54 @@ private void ProcessEvent(WorkspaceChangeEventArgs args, string eventName)
                 switch (args.Kind)
                 {
                     case WorkspaceChangeKind.SolutionAdded:
+                        EnqueueFullSolutionEvent(args.NewSolution, InvocationReasons.DocumentAdded, eventName);
```

Comment:
This is a bit funky that DocumentAdded is being used here; I get the idea that an opening of a solution is nothing more than a bunch of document adds if you squint a bit, but is anything going to assume that DocumetnAdded means there's a DocumentId in the queued item?

Reply:
I have no idea why DocumentAdded is used here. Note that this PR didn't cause this to occur, it's just more obvious now.